documentation/userguide.md

```diff
@@ -95,12 +95,39 @@ After restarting the IIS/recycling the pool and opening the application in a bro
 
 ### Azure Cloud
 
-Azure instantly kills the application when it is stopped/restarted, thus the profiler has no time to write anything to the trace file. To remedy this, enable eager mode (see the Configuration section. Please note that eager mode should only be used together with light mode as it will otherwise drastically increase lag during the application startup.
-
-To deploy the profiler to the Azure cloud, enable the FTP account. Store the profiler in a directory that will not be overwritten by a new deployment, e.g. `D:\home\site\repository\profiler\`
-Also create a new directory to write the traces to, e.g. `D:\home\site\repository\traces\`
-
-To set the environment variables in Azure, go to the application settings and add them as new application settings entries.
+If the application is running in an Azure App Service, the following steps need to be taken:
+
+**Install the Profiler:**
+To deploy the profiler, [enable the FTP account](https://docs.microsoft.com/en-us/azure/app-service/app-service-deploy-ftp) and upload the profiler DLLs to a location that will not be overwritten by a new app deployment, e.g., `D:\home\site\repository\profiler\`.
+To [configure the profiler](#profiler_configuration) in Azure, go to the [application settings](https://docs.microsoft.com/en-us/azure/app-service/web-sites-configure) and add the environment variables as new application settings entries.
```

Comment:
The anchor #profiler_configuration does not exist. please either add or remove the link

Reply:
On GitHub this works, because the headers get Ids and there can be referenced. I can try to explicitly add the anchors. Otherwise I would remove them.

Reply:
Apparently, anchor links are broken in `markdown-pdf` due to [a "bug/feature" in the pdf rendering engine](https://github.com/travs/markdown-pdf/issues/113). I removed the links and reformulated accordingly.